docs/en/reference/basic-mapping.rst

```diff
@@ -239,6 +239,7 @@ Here is a complete list of ``Column``s attributes (all optional):
 - ``nullable`` (default: ``false``): Whether the column is nullable.
 - ``insertable`` (default: ``true``): Whether the column should be inserted.
 - ``updatable`` (default: ``true``): Whether the column should be updated.
+- ``generated`` (default: ``null``): Whether the generated strategy should be ``'NEVER'``, ``'INSERT'`` and ``ALWAYS``.
```

Comment:
not sure if we shold document `NEVER` as for me it doesn't make lot of sense. But it was added to PHPDoc: https://github.com/doctrine/orm/blob/cfc0655a1c9b5ffd5506172cb7642b6090aa5f7e/src/Mapping/Column.php#L18

/cc @mehldau

Reply:
`NEVER` is a valid value so it should be mentioned. A default may change over time.